README.md

```diff
@@ -30,38 +37,158 @@ and uses [lnwallet](https://pkg.go.dev/github.com/lightningnetwork/lnd/lnwallet/
 
 ## Usage Documentation
 
-For a comprehensive guide on how to set up and use Ark, please visit our [Developer Portal](https://arkdev.info).
-For a quick-start with Docker, head over to our [Quick Start guide](https://arkdev.info/docs/quick-start/overview) where you can setup an Ark Server and Clients in a local Bitcoin regtest network.
+Welcome to the `arkd` repository, a Bitcoin server for off-chain Bitcoin transactions using the Ark protocol.
+
+### Installing from GitHub Releases
+
+1. Download the latest `arkd` binary from the [GitHub Releases page](https://github.com/ark-network/ark/releases)
+
+2. Make the binary executable:
+   ```sh
+   chmod +x arkd
+   ```
+
+3. Move the binary to a directory in your PATH (optional):
+   ```sh
+   sudo mv arkd /usr/local/bin/
+   ```
+
+### Configuration Options
+
+The `arkd` server can be configured using environment variables.
+
+| Environment Variable | Description | Default |
+|---------------------|-------------|--------|
+| `ARK_NETWORK` | Bitcoin network (bitcoin, testnet3, regtest, signet, mutinynet) | `bitcoin` |
+| `ARK_DATADIR` | Directory to store data | App data directory |
+| `ARK_PORT` | Port to listen on | `7070` |
+| `ARK_LOG_LEVEL` | Logging level (0-6, where 6 is trace) | `4` (info) |
+| `ARK_ROUND_INTERVAL` | Interval between rounds in seconds | `30` |
+| `ARK_DB_TYPE` | Database type (sqlite, badger) | `sqlite` |
+| `ARK_EVENT_DB_TYPE` | Event database type (badger) | `badger` |
+| `ARK_SCHEDULER_TYPE` | Scheduler type (gocron, block) | `gocron` |
+| `ARK_TX_BUILDER_TYPE` | Transaction builder type (covenantless) | `covenantless` |
+| `ARK_WALLET_ADDR` | Wallet address | - |
```

Comment:
these can be dropped (ARK_WALLET_ADDR will be reintroduced with v0.7)

Reply:
done